### PR TITLE
#qvapay sdk #2

### PR DESCRIPTION
**Pull request by j-fuxa**

### CODE REVIEW

File: QvaPay.Net/Objects/User/UserInfo.cs [hunk 1]
```diff
@@ -20,11 +20,17 @@ public class UserInfo
         public string? Lastname { get; set; }
         [JsonPropertyName("bio")]
         public string? Bio { get; set; }
-        [JsonPropertyName("logo")]
-        public string? Logo { get; set; }
+        [JsonPropertyName("profile_photo_path")]
```
Comment (eyonys) [inline QvaPay.Net/Objects/User/UserInfo.cs:21]: image

El json devuelve el atributo logo

**Comment by j-fuxa** [comment]
In the [0f5a513](https://github.com/qvapay/client-csharp/pull/2/commits/0f5a51309a2e6545e4ddee72519e6fedc71c5772) some modifications to the repo structure was made

- Code was moved to the "src" directory in order to make a cleaner repo root (For adding examples in the future)
- Base Namespace has been renamed to "QvaPay.Sdk" as the ".Net" namespace is usually reserved for network communication and could lead to some confusion.

Other changes are self-explanatory.

File: QvaPay.Net/Objects/User/UserInfo.cs (hunk 1 above)
Comment (j-fuxa) [reply QvaPay.Net/Objects/User/UserInfo.cs:21]: image

Al parecer recibimos respuestas parcialmente diferentes de la Api, alguna idea de a que se debe esto?

Comment (eyonys) [reply QvaPay.Net/Objects/User/UserInfo.cs:21]: 👀Vaya vaya

**Review by eyonys — APPROVED** [review]

**Pull request merged**